### DISCUSSION TITLE
Draft Orders <> Payment Customization Function

### DISCUSSION THREAD

**Discussion by smiddygroup** [Shopify/function-examples#564] [Feedback: Payment Customization API]
Do we know when Draft Orders will be supported via Payment Customization Function API? Seeing it works for other function based methods, but not for Payments.

**Comment by gil--** [comment]
Closing as this is now supported https://changelog.shopify.com/posts/draft-orders-now-support-delivery-and-payment-method-customizations-and-checkout-rules
